src/LSODA.cpp

```diff
@@ -86,11 +86,15 @@ LSODA::LSODA(int neq_, const Rcpp::S4& mod) {
   maxsteps_(Rcpp::as<int>(mod.slot("maxsteps")));
   ixpr_(Rcpp::as<int>(mod.slot("ixpr")));
   mxhnil_(Rcpp::as<int>(mod.slot("mxhnil")));
-  itol_ = 1;
```

Comment:
We used to assume itol was 1; now we have to pass in via the model object and check.